docs/whatsnew/1.0.0/1_0_0.rst

```diff
@@ -0,0 +1,70 @@
+Pylake 1.0.0
+============
+
+.. only:: html
+
+Pylake has made it to `1.0.0`! And with it, several features and improvements have been added.
+
+Documentation
+-------------
+
+.. image:: docs.png
+
+The documentation was completely reworked.
+We separated :doc:`the tutorial section</tutorial/index>`, which teaches you how to use Pylake from :doc:`the theory section</theory/index>`, which teaches you the theory underlying the implemented methods.
+In addition, both sections were extended considerably.
+
+Every tutorial now downloads its own `example data <https://doi.org/10.5281/zenodo.7729525>`_ automatically when you run it.
+This allows you to immediately try out the various functionalities Pylake has to offer.
+The new :doc:`theory section</theory/index>` provides examples and references to literature for further reading.
+
+Creating kymographs from camera recordings
+------------------------------------------
+
+.. image:: kymo_to_stack.png
+
+A new feature introduced in `1.0.0` is the ability to convert camera recordings to kymographs.
+After defining a tether, you can call :meth:`~lumicks.pylake.ImageStack.to_kymo` which rotates the camera images such that the tether is aligned with the horizontal axis and sums a region around the tether.
+To read more about this feature, please refer to the :ref:`tutorial<kymo_from_image>`.
+
+Bias correction
+---------------
+
+When tracking kymographs, the default refinement algorithm used to achieve sub-pixel accuracy is based on centroid refinement.
+Centroid refinement can suffer from bias (tending towards the pixel center) when there is considerable background in the image.
+
+.. figure:: bias_correction.png
+
+    Bias due to high background.
+    On the x-axis is the true position of a fluorescent spot, while on the y-axis, we see its inferred location.
+    From left to right: regular centroid refinement, bias-corrected refinement and Gaussian refinement.
```

Comment:
How about:
```
    Localization in the presence of high background.
    On the x-axis is the true position of a fluorescent spot, while on the y-axis, we see its inferred location.

    From left to right: regular centroid refinement, bias-corrected refinement and Gaussian refinement.
    The step-like progression in the centroid case is due to the bias induced by the background intensity.
    These steps originate from a bias towards the center of each pixel.
    Note how the bias corrected variant more closely agrees with the Gaussian refinement result.

    From top to bottom: The top row shows localization performed with a small window, while the bottom row shows localization performed with a small window.
    For Gaussian refinement, a larger window is beneficial (as it uses more data and models the background), while for centroid refinement it can lead to a larger estimation variance because of the inclusion of more noise.
```
for the entire figure legend?